localization/english/national_focus_l_english.yml

```diff
@@ -1,38 +0,0 @@
-﻿l_english:
-decree_road_maintenance:0 "Efficient Administration"
-decree_road_maintenance_desc:0 "Encourage efficient state administration"
```

Comment:
where did this go?

Reply:
Oh! Didn't see that one had been renamed, just a second.